modules/deployment/s3.tf

```diff
@@ -0,0 +1,19 @@
+module "s3_bucket" {
```

Comment:
the acl is `private` by default, but are there, from a security perspective, any other values we should set (e.g `block_public_acls` and `block_public_policy`?

Reply:
`private` means that only the owner can access those files. with the `block_*` you can additionally block put requests that would violate this acl.

but since this is only a "technical bucket for artifact storage", no user should ever actually interfere with its contents.